android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/model/LinkTapEvent.kt

```diff
@@ -29,15 +29,15 @@ class LinkTapEvent(
     val documentX: Float,
     val documentY: Float,
     val mappedLinkRect: RectF,
-    link: PdfDocument.Link
+    link: Link
 ) {
-    private val link: PdfDocument.Link
+    private val link: Link
 
     init {
         this.link = link
     }
 
-    fun getLink(): PdfDocument.Link {
+    fun getLink(): Link {
         return link
```

Comment:
This is really weird just make `link` property a val inside the constructor to get rid of extra setter/getter